cmd/admin-router.go

```diff
@@ -143,4 +143,5 @@ func registerAdminRouter(router *mux.Router, enableConfigOps, enableIAMOps bool)
 
 	// If none of the routes match, return error.
 	adminV1Router.NotFoundHandler = http.HandlerFunc(httpTraceHdrs(notFoundHandlerJSON))
```

Comment:
Also, do we need a NotFoundHandler()?

Reply:
It was not called in the first place. We could remove this IMO.

Reply:
Please do.. that is better.